types/jest/index.d.ts

```diff
@@ -153,6 +153,11 @@ declare namespace jest {
      * for the specified module.
      */
     function setMock<T>(moduleName: string, moduleExports: T): typeof jest;
+     /**
+     * Set the default timeout interval for tests and before/after hooks in milliseconds.
+     * Note: The default timeout interval is 5 seconds if this method is not called.
+     */
+    function setTimeout(timeout: number): typeof jest
```

Comment:
Can you add a test for it? e.g. `jest.setTimeout(1000);`.

And you have to fix the linter error.